lib/backend.js

```diff
@@ -91,6 +91,19 @@ class Backend {
   processInfo() {
     return this._info;
   }
+
+  async loadNetworkResource(url, headers) {
+    try {
+      if (url.startsWith('file://')) {
```

Comment:
Is this only used by trusted code (e.g. no concerns regarding reading `/etc/passwd`)?

Reply:
User can start child process with `process.cwd() === '/etc'` or evaluate `fs.readFileSync('/etc/passwd')` from console. I do not have any security assumptions inside `ndb`.